VideoLocker/assets/config/registration_form.json

```diff
@@ -35,7 +33,6 @@
         "errorMessages": {},
         "placeholder": "",
         "defaultValue": "",
```

Comment:
We talked about this a few weeks ago, but if I recall @lizcohen and @scottrish were part of that conversation. I think they thought that the extra information wasn't necessary.

@lizcohen? What are your thoughts?